lib/did-veres-one.js

```diff
@@ -170,6 +170,47 @@ api.generate = util.callbackify(async function(
   return {publicDidDocument, privateDidDocument};
 });
 
+/**
+ * Convert a private key to a public key
+ */
+api.publicDidDocument = util.callbackify(async ({privateDidDocument}) => {
+  // whitelist copy of public details
+  const publicDidDocument = {
+    '@context': privateDidDocument['@context'],
+    id: privateDidDocument.id
+  };
+  const suites = ['authentication', 'grantCapability', 'invokeCapability'];
+  for(const suite of suites) {
+    publicDidDocument[suite] = [];
+    for(const item of privateDidDocument[suite]) {
+      const pubItem = {
+        type: item.type,
+        publicKey: []
+      };
+      for(const field of ['@context']) {
```

Comment:
Is there a reason why this is a loop?

Reply:
I think there were some other items in that array that got taken out.